ddtrace/tracer/option.go

```diff
@@ -235,6 +235,9 @@ type config struct {
 	// partialFlushEnabled specifices whether the tracer should enable partial flushing. Value
 	// from DD_TRACE_PARTIAL_FLUSH_ENABLED, default false.
 	partialFlushEnabled bool
+
+	// enableDataStreamsMonitoring enables monitoring of data streams
+	enableDataStreamsMonitoring bool
```

Comment:
nit:
Let's call this `dataStreamsMonitoringEnabled`. Most of the others are formatted that way, except for `enableHostnameDetection`, which I feel was an oversight.